http-watch.sh

```diff
@@ -38,6 +38,7 @@ then
 	mkdir -p $LOG_PATH # If it does not, create the path from the config
 fi
 
+DATE # Get the date
```

Comment:
Likewise - I think this function (which has the same name as a command) might only be called upon once. It may be simpler and more self-explanatory to just have the full date command in situ in the code rather than having it burried away within a variable, within a function, both of which may be unnecessary.

functions should be saved for times where your going to need that function / operation again and again, but if its a once or twice only style thing then it may not warrant it.

Reply:
I have re-visited this particular function and have removed it from the latest commit following your advice and feedback from other points.